driver-scala/src/main/scala/org/mongodb/scala/model/Aggregates.scala

```diff
@@ -17,12 +17,14 @@
 package org.mongodb.scala.model
 
 import com.mongodb.client.model.fill.FillOutputField
+
 import scala.collection.JavaConverters._
-import com.mongodb.client.model.{ Aggregates => JAggregates }
+import com.mongodb.client.model.{ Aggregates => JAggregates, GeoNearOption }
```

Comment:
No need to import `GeoNearOption` as you have the type alias defined in `org.mongodb.scala.model.package.scala`